deployment-templates/deploy-forseti.yaml.in

```diff
@@ -13,7 +13,9 @@
 # limitations under the License.
 
 # The toplevel deployment script for Forseti Security.
+# Be sure to edit this file and its templates to suit your environment.
```

Comment:
Remove this line, this file is automatically updated, not meant to be edited by hand